app/Cart.php

```diff
@@ -70,10 +70,10 @@ public function removeItem($item)
     public function updateQuantity($item, $qty)
     {
         // Count the number of $items in array
-        $filteredItems = array_filter($this->items, function($value) use ($item) { return $value === $item; });
```

Comment:
So `array_filter` will return an _entirely new_ array, it doesn't modify the existing one.

...so if you do:

```php
$myArray = [1, 2, 3];

array_filter($myArray, function($e) { return $e !== 1 });

print_r($myArray);
```

...what do you see?